core/src/test/java/com/graphhopper/reader/OSMReaderTest.java

```diff
@@ -648,7 +648,7 @@ public long handleNodeTags( OSMNode node )
         OSMWay way = new OSMWay(1L);
         way.getNodes().add(1);
         way.getNodes().add(2);
-        way.setTag("highway", "motorway");
+        way.setTag("route", "ferry");
```

Comment:
The test is testing the calculated estimated_center. This artificially created tag is not used anywhere in the productive code, just in tests.